### ISSUE TITLE
Do we have support for prime-ng tags as well?

### ISSUE THREAD

**Issue by gururaj1982** [mvdschee/web-accessibility#32]
In my HTML page I use a mix of HTML and Prime NG tags , I see that html accessibility issues are highlighted, but not prime ng tags. Is there a plan to support prime-ng tags as well.
Thanks in advance

**Comment by mvdschee** [comment]
He @gururaj1982 That's a tough one to say because I would love to support as much as possible, but what they have is a UI Component Libraries which should have most rules covered. I see they are "Fully accessible and in compliance with Section 508 standards." which is sort of equal to WCAG 1.0 and more as a basic ruleset. So for me to write exceptions for that library would mean making rules more loosely for the sake of support. I don't feel like that would be a step in the right direction, but I'm open for a discussion :)
